operator/controllers/mlops/server_controller.go

```diff
@@ -148,7 +160,7 @@ func serverReady(status mlopsv1alpha1.ServerStatus) bool {
 }
 
 func (r *ServerReconciler) updateStatusFromError(ctx context.Context, logger logr.Logger, server *mlopsv1alpha1.Server, err error) {
-	server.Status.CreateAndSetCondition(mlopsv1alpha1.StatefulSetReady, false, err.Error())
+	server.Status.CreateAndSetCondition(mlopsv1alpha1.DeploymentReady, false, err.Error())
```

Comment:
isn't this code used by both StatefulSets and Deployments? (same question for the next change in this file)